src/main/java/org/jitsi/videobridge/RtpChannel.java

```diff
@@ -657,7 +657,7 @@ private boolean addReceiveSSRC(int receiveSSRC, boolean checkLimit)
     @Override
     protected void closeStream()
     {
-        if (!streamClosed)
+        if (!streamClosed && stream != null)
         {
```

Comment:
What if stream is not closed, but stream is null. Should we always set `streamClosed = true;`

Reply:
I don't think that it matters. The `streamClosed` flag is only used to prevent calling `close` twice.